import-js.py

```diff
@@ -136,6 +136,8 @@ def handle_resolved_imports(resolved_imports):
                 args['command'] = 'add'
                 args['imports'] = resolved_imports
                 self.run(edit, **args)
+            self.view.run_command("import_js_replace",
+                                  {"characters": result.get('fileContent')})
             self.ask_to_resolve(result.get('unresolvedImports'),
                                 handle_resolved_imports)
             return
```

Comment:
I think I've seen that issue too. Despite the async look (passing a callback) things are actually run in sync.